test/index.js

```diff
@@ -65,6 +68,19 @@ describe('hubdown', () => {
     file.content.should.include('<em>Markdown</em>')
   })
 
+  describe('ignoreMissing', () => {
+    it('throw an error when unknown language is present', () => {
+      return hubdown(fixtures.unknownLanguage).should.to.be.rejectedWith(/Unknown language: `the-unknown-language-whats-actually-graphql`/)
+    })
+
+    it('should work when ignoreMissing on the scene', async () => {
+      const file = await hubdown(fixtures.unknownLanguage, { ignoreMissing: true })
+      $ = cheerio.load(file.content)
+      fixtures.unknownLanguage.should.include('```the-unknown-language-whats-actually-graphql')
```

Comment:
```suggestion
      fixtures.unknownLanguage.should.include('```some-unknown-language')
```